crates/uv-cli/src/lib.rs

```diff
@@ -244,7 +244,48 @@ pub enum Commands {
         after_long_help = ""
     )]
     Tool(ToolNamespace),
-    /// Manage Python versions and installations (experimental).
+
+    /// Manage Python versions and installations (experimental)
```

Comment:
Intentional that period was removed here?

Reply:
Yes, I switched it to verbatim mode which I think would retain the period but clap strips them everywhere else. We need to drop the period for consistency. Kind of awkward, https://github.com/clap-rs/clap/issues/3367